cmd/habitat/community/executor.go

```diff
@@ -126,3 +128,24 @@ func (e *CommunityExecutor) StopProcessInstance(update *state.StateUpdate) error
 
 	return nil
 }
+
+func (e *CommunityExecutor) AddNode(update *state.StateUpdate) error {
```

Comment:
should this be `AddIPFSNode`? since not all nodes will run IPFS? or is that wrong

Reply:
This function is tied to the `AddNode` state update. It will eventually have behavior thats also not IPFS related when the `libp2p` PR gets in

Reply:
so right now we expect that all nodes run IPFS right?